datakyt/db/create_database.sql

```diff
@@ -0,0 +1,5 @@
+CREATE DATABASE datakyt;
+
+CREATE USER datakyt_admin WITH ENCRYPTED PASSWORD 'password';
+
+GRANT ALL PRIVILEGES ON DATABASE datakyt TO datakyt_admin;
```

Comment:
I think we should add such a line 
`GRANT ALL ON SCHEMA public TO postgres;` 
to create_database script so `postgres` will be another super-user

Reply:
@paulzcooper let's skip this for now and add once we see it's needed.

Reply:
@aartiukh okay